api-reference/10 UI Components/dxChart/1 Configuration/argumentAxis/breaks/breaks.md

```diff
@@ -14,7 +14,7 @@ A scale break allows breaking off a part of the axis to improve the readability
 
 ![DevExtreme HTML5 JavaScript Charts Scale Breaks](/images/ChartJS/visual_elements/scale-breaks_arg-axis.png)
 
-Each object in the **breaks** array configures a single scale break. Note that a scale break is visible only if its range exceeds the tick interval.
+Each object in the **breaks** array configures a single scale break. We recommend that a scale break range exceeds the tick interval twice. Otherwise, the scale break might not be visible.
```

Comment:
```suggestion
Each object in the **breaks** array configures a single scale break. We recommend that a scale break range exceeds the tick interval at least twice. Otherwise, the scale break might not be visible.
```